pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = --cov . --cov-config .coveragerc --pep8 --pylint --cov-report term --cov-report html --ds=ccxcon.settings
+pep8ignore =
```

Comment:
I'd like to propose we up our line length limit to 100 characters. There is support for this in pep8 itself.

http://legacy.python.org/dev/peps/pep-0008/#maximum-line-length